MinecraftForge/MinecraftForge
# Re-add hook for LivingExperienceDropEvent, fixes #7900

Re-adds the hook that got missed during the 1.17 port.

Closes #7900

Testing:
----- 
- Enable Test Mod
- Join World
- Spawn a `pig` and kill it
- Verify the logged message
- Verify via `/xp query points` that the received xp was double the reported original.
